mail_tracking/models/mail_thread.py

```diff
@@ -30,43 +30,62 @@ def _get_failed_message_domain(self):
     def message_post(self, *args, **kwargs):
         """Adds CC recipient to the message.
 
-        Because Odoo implementation avoid store cc recipients we ensure that
-        this information its written into the mail.message record.
+        Because Odoo implementation avoid store 'from, to, cc' recipients we
+        ensure that this information its written into the mail.message record.
         """
-        new_message = super().message_post(*args, **kwargs)
-        email_cc = kwargs.get('cc')
-        if email_cc:
-            new_message.sudo().write({
-                'email_cc': email_cc,
-            })
-        return new_message
+        kwargs.update({
```

Comment:
Adding these 2 keyword arguments doesn't include them in the `mail.message` resulting record. You have to pass them by context and override `mail.message` create method for injecting them on `vals` before calling `super`.

Reply:
This works because the parent method do this: https://github.com/odoo/odoo/blob/12.0/addons/mail/models/mail_thread.py#L2047

** Tested sending a email and works.

Reply:
OK, I see, great!